8/SiloHost/src/Program.cs

```diff
@@ -9,70 +8,29 @@
 using Orleans.Configuration;
 using Orleans.Hosting;
 using Orleans.Statistics;
-using System;
-using System.Net;
-using System.Net.NetworkInformation;
-using System.Net.Sockets;
 using System.Threading.Tasks;
+using Amazon.Util.Internal;
 
 namespace SiloHost
 {
     class Program
     {
+        private static readonly EnvironmentVariables environmentVariable = new EnvironmentVariables();
+        private readonly string EcsContainerMetadataUri = environmentVariable.EcsContainerMetadataUri();
```

Comment:
Let's not retrieve it here. Only when we are 100% sure we don't want to run this locally.

Reply:
That's a leftover, don't actually use it